prefect_gcp/cloud_run.py

```diff
@@ -678,6 +691,7 @@ def _jobs_body(self) -> dict:
                             "spec": {
                                 "containers": containers,
                                 "timeoutSeconds": timeout_seconds,
+                                "maxRetries": max_retries,
```

Comment:
can we omit the above if we do this? it doesn't seem to be used elsewhere
```suggestion
                                "maxRetries": self.max_retries,
```

Reply:
Sure, done :)